distribution/lib/Standard/Table/0.0.0-dev/src/Data/Column.enso

```diff
@@ -1159,7 +1176,7 @@ type Column
     ## PRIVATE
        Returns a column with a continuous sub-range of rows taken.
     slice : Integer -> Integer -> Column
-    slice self start end =
+    slice self start=0 end=(self.length-1) =
```

Comment:
This is a private helper function, are you sure we want defaults on it?

This is a matter of style and so a bit subjective, my personal taste is in cases like this one to avoid defaults to instead make it more clear how the helper is used - otherwise it becomes a bit implicit which IMO can lead to mistakes.

Also why make the default `end` `self.length-1` instead of `self.length`? Sounds a bit counter-intuitive. Unless we have changed `end` to be inclusive - but I'm pretty sure it has been exclusive in all other variants of slice we had (like `Vector.slice`) so that change would be very unexpected.